drivers/utils_test.go

```diff
@@ -0,0 +1,36 @@
+package drivers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestUnescapeUnicodeSequencesFromURL(t *testing.T) {
+	testData := []struct {
+		name      string
+		url       string
+		expected  string
+		expectErr bool
+	}{
+		{
+			name:      "should unescape the escape sequences",
```

Comment:
Can we add a postgres test as well? `postgres://mmuser:mostest@localhost/loadtest?sslmode=disable\u0026binary_parameters=yes`